docs/developer/extending/apps/developing-apps/troubleshooting.mdx

```diff
@@ -29,3 +29,38 @@ The issue might be caused by the CORS policy set up in the Saleor Cloud dashboar
 
 - Add your application URL to the list of allowed origins.
 - Ensure your application makes GraphQL requests from the server side only.
+
+## Inspecting webhook delivery
+You can query app logs to see webhooks' delivery attempts.
+```graphql
+{
+  apps(first:2){
+    edges{
+      node{
+        type
+        identifier
+        webhooks{
+          eventDeliveries(first:10){
+            edges{
+              node{
+                attempts(first:10){
+                  edges{
+                    node{
+                      responseStatusCode
+                      response
+                      responseHeaders
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+Alternatively, you can also see a log of webhook deliveries in the `Dashboard -> Apps -> Your app -> Manage App` section.
```

Comment:
Worth adding note:
If your subscription requires permissions, that are not granted to your app, delivery attempts will not be included in the results of this query.